compiler/src/iree/compiler/Codegen/LLVMGPU/test/BUILD.bazel

```diff
@@ -69,6 +69,7 @@ iree_lit_test_suite(
             "transform_vector_to_mma.mlir",
             "transpose_pipeline_test.mlir",
             "configure_tensor_layout.mlir",
+            "config_reduction_vector_distribute.mlir",
```

Comment:
I think there are tests in LLVMGPU/test/ROCDL . Let's add them in the same file for now. We can improve it later.